scripts/eurostat/gdp/eurostat_gdp.mcf

```diff
@@ -0,0 +1,14 @@
+Node: dcid:Amount_EconomicActivity_GrossDomesticProduction
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+statType: dcs:measuredValue
+
+Node: dcid:Amount_EconomicActivity_GrossDomesticProduction_Count_Person
```

Comment:
Amount_EconomicActivity_GrossDomesticProduction_Count_Person -> Amount_EconomicActivity_GrossDomesticProduction_AsAFractionOfCount_Person

(No need to reimport, since the Pop Obs MCF is already generated and independent now. But updating this and the tmcf will help future imports have consistent localCuratorLevelIDs.)

Reply:
Due to the 2 extra nodes in tmcf (value geo and time) and P1M, we do actually need a reimport.

Reply:
Done.

Reply:
I'm still not seeing "Amount_EconomicActivity_GrossDomesticProduction_AsAFractionOfCount_Person" -- am I viewing the changes incorrectly? I see the change in TMCF.. perhaps the change to MCF is still local?

Reply:
Sorry I had changed this in the tmcf and accidentally skipped mcf changes. Now everything should be good.